### PR TITLE
Add RGB command

### PR DESCRIPTION
**Pull request by lightcodeblocks**
Implements #21

### CODE REVIEW

File: bot/extensions/fun/fun_cog.py [hunk 1]
```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
...
+    async def color_command(self, ctx, R: int, G: int, B: int):
+        rgb = (R, G, B)
+        img = Image.new('RGB', (200, 200), rgb)
+
+        img.save('color.png')
+        embed = Embed(
+            color=self.bot.default_color,
+            description='Here goes your color!'
+        )
+        embed.set_image(url="attachment://filename.png")
```
Comment (PenguinBoi12) [inline bot/extensions/fun/fun_cog.py:54]: If I'm not mistaken, filename needs to be the actual file name so, in this case, `color.png` and not `filename.png` [Discord.py doc about that](https://discordpy.readthedocs.io/en/latest/faq.html#how-do-i-use-a-local-image-file-for-an-embed-image ). 

Last thing is, is this image saved locally or is it just temporarily saved? If it's saved locally on the computer, be sure it's deleted afterward.

File: bot/extensions/fun/fun_cog.py [hunk 2]
```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
...
+        file = File('color.png')
```
Comment (PenguinBoi12) [inline bot/extensions/fun/fun_cog.py:55]: Same thing here, I don't know if it's really necessary to assign it to `file` but it's optional. I don't mind if it's stay like this. :)

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
Overall it's awesome. It's a really cool and fun feature and seems to be working quite well. However, there's is a few thing that I'd like to be adjusted (I know I'm annoying haha). Also, if you ever want to add more "type" of color like hexa and stuff like this (like the issue propose), it would be awesome but it's optional of course.

Thank you for this contribution, you of course win a special role (**contributor**) on discord and as soon, which I doubt will be long, as it's accepted you'll get (**code contributor**). Of course, if you have any question, don't hesitate to simply answer to the comments on your code or simply ask on discord. :)

File: bot/extensions/fun/fun_cog.py [hunk 3]
```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
+    async def color_command(self, ctx, R: int, G: int, B: int):
+        rgb = (R, G, B)
```
[the remark this answers is not in the document]
Comment (lightcodeblocks) [reply bot/extensions/fun/fun_cog.py]: Fixed, thanks! It was actually a leftover from when I was using `*args` as the argument.

File: bot/extensions/fun/fun_cog.py (hunk 1 above)
Comment (lightcodeblocks) [reply bot/extensions/fun/fun_cog.py]: The image seems to show just by attaching a file (without `set_image`), so I just removed those lines!

File: bot/extensions/fun/fun_cog.py (hunk 2 above)
Comment (lightcodeblocks) [reply bot/extensions/fun/fun_cog.py:54]: I think it's good practice to initialize objects separately maybe.
Debatable, but maybe it'll stay like this :P

File: bot/extensions/fun/fun_cog.py [hunk 4]
```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
+    async def color_command(self, ctx, R: int, G: int, B: int):
```
[the remark this answers is not in the document]
Comment (lightcodeblocks) [reply bot/extensions/fun/fun_cog.py]: Fixed. Thanks for noticing!

File: bot/extensions/fun/fun_cog.py [hunk 5]
```diff
@@ -39,6 +41,20 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
```
[the remark this answers is not in the document]
Comment (lightcodeblocks) [reply bot/extensions/fun/fun_cog.py:45]: I couldn't find a usage parameter in the command decorator. Maybe I'm not looking at the right docs?

**Comment by lightcodeblocks** [comment]
Just added a hex command as well!

File: bot/extensions/fun/fun_cog.py (hunk 2 above)
Comment (PenguinBoi12) [reply bot/extensions/fun/fun_cog.py:52]: I wouldn't say it's "good" practice exactly but it's for sure isn't bad so it's fine.

File: bot/extensions/fun/fun_cog.py [hunk 6]
```diff
@@ -39,6 +42,36 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    async def display_color(self, color_img, ctx):
```
Comment (PenguinBoi12) [inline bot/extensions/fun/fun_cog.py:45]: It's very good that you separated this logic from the commands.

File: bot/extensions/fun/fun_cog.py [hunk 7]
```diff
@@ -39,6 +42,36 @@ async def quote_command(self, ctx):
...
+            color=self.bot.default_color,
+            description='Here goes your color!'
+        )
+        file = File('color.png')
+        await ctx.send(embed=embed, file=file)
+
+        os.remove('color.png')
+    
+
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
```
Comment (PenguinBoi12) [inline bot/extensions/fun/fun_cog.py:58]: You don't need the three `'''`. You can just use single quote (') or double quotes (")

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
It's very good. I have one idea though. I think it would make more sense to put all that in it's own cog (extension). Like that, you could even make the commands part of a group so calling them would look like this `@Grace color hex ff00ff` which will make it even clearer that we generate a color from an hexadecimal value and also a bit more structured.

File: bot/extensions/fun/fun_cog.py (hunk 7 above)
Comment (lightcodeblocks) [reply bot/extensions/fun/fun_cog.py]: Fixed!

**Review by PenguinBoi12 — APPROVED** [review]
Congrats. I added the usage and removed a forgotten extra line but now it seems good so it's ready to be merged. Well done!

**Pull request merged**